projects/mind-restake/index.js

```diff
@@ -1,22 +1,28 @@
 const config = {
-  ethereum: "0x6a5D488EC17d6a7a1872AaB88feC90c1B2Df4196",
-  scroll: "0xea3E87699D11B77Fba754Bf0257a25664B97437d",
-  bsc: "0x6a5D488EC17d6a7a1872AaB88feC90c1B2Df4196",
+  bsc: "0xb080B94052f039eC2CA8BBaF7Ec13329d1926973",
+  fhe:"0xdeD96288c99145da4800f55355A2466f6238fBBE"
 };
 
+const token = "0xd55C9fB62E176a8Eb6968f32958FeFDD0962727E"
+
 module.exports = {
-  methodology: "Counts the total amount of asset tokens deposited in each of the Strategy contracts registered in the helper contract on each chain.",
+  methodology: "Counts the total amount of FHE tokens deposited in Mind Agentic World contracts on BSC and MindChain.",
 }
 
 Object.keys(config).forEach(chain => {
   const target = config[chain]
   module.exports[chain] = {
     tvl: async (api) => {
-      const [_vaults, tokens, bals] = await api.call({
-        abi: "function getPoolTotalAssets() view returns (address[] memory,address[] memory, uint256[] memory)",
-        target,
-      })
-      api.add(tokens, bals)
+      const tvlAmount =  await api.call({
+        abi: 'erc20:balanceOf',
+        target: token,
+        params: [target],
+      }); 
+       const bigIntAmount = BigInt(tvlAmount); 
+       const amountInToken = bigIntAmount / BigInt(10 ** 18); 
+      // api.add(token, tvlAmount,{ priceChain: "bsc" })
+      api.addCGToken("mind-network", amountInToken)
```

Comment:
please export any TVL made up of your own token FHE under a `staking` key rather than `tvl`